README.md

```diff
@@ -1,2 +1,21 @@
+Humanoid League Messages
+========================
+
+Standard messages:
+------------------
+
+Standard messages are used for:
+
+- ball_relative (PoseWithCovariance)
+- balls_relative (PoseWithCovarianceArray)
```

Comment:
I did not found any `PoseWithCovarianceArray` in the source code of https://github.com/ros/common_msgs, therefore I think, this message is not a standard nor anywhere defined...

We could define an own `PoseWithCovarianceArrayStamped` to solve this and the issue below with `goalposts_relative`.

Reply:
I will do this. But it will be called `PoseWithCovarianceArray` as the original PoseArray is also stamped by default.